ts/src/static_dependencies/jsencrypt/lib/jsrsasign/asn1-1.0.d.ts

```diff
@@ -1,77 +1,25 @@
-
-
-declare interface IDERIntegerParams {
```

Comment:
@ttodua u sure it is ok to remove this?

Reply:
ohh, i have pushed several times and missed i had pushed this file too. it shouldnt have been pushed. reverted